src/input/input.style.tsx

```diff
@@ -96,14 +97,15 @@ export const Container = styled.div<InputStyleProps>`
 `;
 
 export const InputElement = styled.input<InputStyleProps>`
+    ${TextStyleHelper.getTextStyle("Body", "regular")}
+
     // overwrite default styles
     background: transparent;
     border: none;
     height: 3rem;
     width: 100%;
 
-    ${TextStyleHelper.getTextStyle("Body", "regular")}
-    padding: 0.2rem 1rem 0.3rem; // Somehow the input text appears lower
+    padding: 0 0 0.1rem; // Somehow the input text appears lower
```

Comment:
use the parent to set the padding/margins rather than the input to give more flexibility